### PR TITLE
Public Screen

### PR DESCRIPTION
**Pull request by ralucachis-vsp**
This PR contains:
- Public Screen header and tabs
- Archive name in the toolbar

### CODE REVIEW

File: app/src/main/java/org/permanent/permanent/ui/public/PublicArchiveFragment.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package org.permanent.permanent.ui.public
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.permanent.permanent.databinding.FragmentPublicArchiveBinding
+import org.permanent.permanent.ui.PermanentBaseFragment
+
+class PublicArchiveFragment : PermanentBaseFragment() {
```
Comment (cgeorgiuvsp) [inline app/src/main/java/org/permanent/permanent/ui/public/PublicArchiveFragment.kt:10]: is this finalized? Do we have a viewmodel for this view?

File: app/src/main/java/org/permanent/permanent/ui/public/PublicFragment.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          binding.lifecycleOwner = this
+          binding.viewModel = viewModel
+
+        (activity as AppCompatActivity?)?.supportActionBar?.title = viewModel.getCurrentArchiveName()
+
+          return binding.root
+      }
+
+    val tabArray = arrayOf(
+        "Public Archive",
```
Comment (cgeorgiuvsp) [inline app/src/main/java/org/permanent/permanent/ui/public/PublicFragment.kt:36]: use string resources

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -115,6 +116,32 @@
...
+    <string name="public_profile_read_more">Read More</string>
+    <string name="public_profile_person_information">Person Information</string>
+    <string name="public_profile_full_name_hint">Full name</string>
+    <string name="public_profile_birth_date">Birth Date</string>
+    <string name="public_profile_birth_date_hint">YYYY-MM-DD</string>
+    <string name="public_profile_birth_location">Birth Location</string>
+    <string name="public_profile_birth_location_hint">Choose a location</string>
+    <string name="public_profile_online_presence">Online Presence</string>
+    <string name="public_profile_milestones">Milestones</string>
+    <string name="public_profile_about_text">Dr. Robert Friedman has over a decade of experience
```
Comment (cgeorgiuvsp) [inline app/src/main/res/values/strings.xml:131]: are these placeholders? this should not be included in the release

Comment (ralucachis-vsp) [reply app/src/main/res/values/strings.xml:131]: yes, those are placeholders for Public Profile, I will take them out

File: app/src/main/res/mipmap-anydpi-v26/ic_camera.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (flaviahandrea-vsp) [inline app/src/main/res/mipmap-anydpi-v26/ic_camera.xml:1]: is this file used ?

File: app/src/main/res/mipmap-anydpi-v26/ic_camera.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (flaviahandrea-vsp) [inline app/src/main/res/mipmap-anydpi-v26/ic_camera.xml:2]: is this file used ?

File: app/src/main/res/mipmap-anydpi-v26/ic_camera_round.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (flaviahandrea-vsp) [inline app/src/main/res/mipmap-anydpi-v26/ic_camera_round.xml:1]: is this file used ?

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -15,6 +15,7 @@
     <color name="whiteTransparent">#80FFFFFF</color>
     <color name="superLightGrey">#EBEBEB</color>
     <color name="lightGrey">#B4B4B4</color>
+    <color name="lighterGrey">#DADADA</color>
```
Comment (flaviahandrea-vsp) [inline app/src/main/res/values/colors.xml:18]: do we really need this? we have "superLightGrey"

**Review by flaviahandrea-vsp — CHANGES_REQUESTED** [review]

**Pull request merged**